### DISCUSSION TITLE
Can you add a Species ie Kestrel ?

### DISCUSSION THREAD

**Discussion by pomgonewalkabout** [mcguirepr89/BirdNET-Pi#564] [Q&A] [answered]
Hi I am new to using BirdNet-Pi put I have it running on a Raspberry Pi 4 and I am blown away by how it works !
I live in Australia and where I live we have Kestrels, lots of them and nesting too on the property where I live.
So far Birdnet Pi has not recorded one Kestrel even though they are calling out.
Is there a way to add a species to the database such as the Nankeen Kestrel?

Cheers PGW.

**Comment by mcguirepr89** [comment, marked as answer]
Hi, @pomgonewalkabout --

I don't train the model and the model is deprecated, so there won't be any additional data trained into the model.

With that said, I checked and there are three Kestrel species trained in the model:
```
Falco naumanni_Lesser Kestrel
Falco sparverius_American Kestrel
Falco tinnunculus_Eurasian Kestrel
```

I suggest using the log view to see if there are any lower-confidence predictions for those species that would allow you to adjust your minimum confidence score in order to obtain those calls. You may also consider putting together a custom species list if you are interested in specific species, as this would allow you a much lower confidence score for the system overall without (hopefully) too many spurious false-positives.
